runconfig/parse.go

```diff
@@ -96,6 +97,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 	)
 
 	cmd.Var(&flAttach, []string{"a", "-attach"}, "Attach to STDIN, STDOUT or STDERR")
+	cmd.Var(&flBlkioWeightDevice, []string{"-blkio-wight-device"}, "Block IO weight (relative weight for device)")
```

Comment:
typo blkio-**wight**-device should be **weight**
`flag provided but not defined: --blkio-weight-device`